BothamNetworking/BothamAPIClientError.swift

```diff
@@ -8,12 +8,32 @@
 
 import Foundation
 
-public enum BothamAPIClientError: ErrorType {
+public enum BothamAPIClientError: ErrorType, Equatable {
 
     case HTTPResponseError(statusCode: Int, body: NSData)
     case NetworkError
     case HTTPClientError(error: NSError)
     case ParsingError(error: NSError)
     case UnsupportedURLScheme
+    case RetryError
 
 }
+
+public func == (lhs: BothamAPIClientError, rhs: BothamAPIClientError) -> Bool {
```

Comment:
We can't we are using it inside BothamAPIClient as I said in the first PR comment.